chef/data_bags/crowbar/bc-template-provisioner.json

```diff
@@ -79,6 +79,11 @@
       "windows": {
         "admin_password": "crowbar"
       },
+      "suse": {
+        "autoyast": {
+          "packages": [ ]
+        }
+      },
```

Comment:
Why should that conflict? We just realized that @aspiers did no migration script to create the autoyast namespace so that @thutterer have to do it in his migration script.